### ISSUE TITLE
Generate polygons for rings

### ISSUE THREAD

**Issue by lfunkhouser** [Esri/distance-direction-addin-dotnet#625]
_From @sindizzy on August 15, 2018 0:49_

As always thank you so much for a great product!
When we generate ellipses, circles, range rings I would vote for them to be created as a polygon rather than a polyline. Quite often I need to add a fill to the ellipse and as a polyline you can't do that. I believe the reason for this was so all geometry could be saved to one feature class. I think, as a user I would not mind one feature class for polylines and one feature class for polygons. Another reason I think polygons are a good idea is that post-processing like unionization of multiple polygons is possible.

_Copied from original issue: Esri/military-tools-desktop-addins#65_

**Comment by topowright** [comment]
Giving the user the ability to choose the geometry output for circles, ellipses, and polygons could be the path forward for this issue. For ease of use each ring would be a single feature. No unionization of features would happen automatically.

**Comment by sindizzy** [comment]
That's great.

**Comment by dfoll** [comment]
This is something that will make #438 doable as well

**Comment by sindizzy** [comment]
Yes my use case is aviation. We generate what's called Area of Operations circles and other similar features. They essentially show a geoelllipse from a center (an airport or a waypoint) at a specified distance (typically a radius in NM). We then use a fill color to denote where operations are allowed. The fill makes them stand out as opposed to just a perimeter.

**Comment by dfoll** [comment]
@saip is awaiting response about what final implementation would look like for this fix. I think we may need to do better design on this one before we hand it over

**Comment by dfoll** [comment]
@sindizzy your use case is pretty clear. One question for you. In your case is there a requirement to continue to draw rings as polylines? The change we are considering in this case is to switch rings entirely from polyline to polygon instead of offering an option to draw rings as polylines or polygons

**Comment by sindizzy** [comment]
For my case I generate range rings only as polygons. This is something that the Military Analyst did before it was sunsetted. In the past we did generate the rings as polylines **ONLY** because you couldn't label the polygon on the perimeter. The workaround was to generate them as polylines. With Maplex now being part of ArcMap and possibly Pro there is no need for me to create the rings as polylines.

**Comment by dfoll** [comment]
@sindizzy thanks for the response. Good feedback to know you need to be able to label on the perimeter of the polygons. Will keep this in mind as we proceed with design for this

**Comment by sindizzy** [comment]
Can we get this in the ArcMap extension as well?
